### PR TITLE
feat(core): rename send_usage into send_telemetry

### PR DESCRIPTION
**Pull request by QuentinBrosse**

### CODE REVIEW

File: scw/config_test.go [hunk 1]
```diff
@@ -652,7 +652,7 @@ func TestConfig_ConfigFile(t *testing.T) {
 
 # To improve this tools we rely on diagnostic and usage data.
```
Comment (kindermoumoute) [inline scw/config_test.go]: `to improve the CLI`

Comment (kindermoumoute) [reply scw/config_test.go]: Or add a note `This attribute is only used for the CLI`

Comment (kindermoumoute) [reply scw/config_test.go]: `we rely on diagnostic and usage data.` ==> `we rely on usage telemetry.`

File: scw/config_test.go [hunk 2]
```diff
@@ -652,7 +652,7 @@ func TestConfig_ConfigFile(t *testing.T) {
...
 # Sending such data is optional and can be disable at any time by setting this variable to false.
```
Comment (kindermoumoute) [inline scw/config_test.go]: Sending telemetry is optional..

File: scw/config.go [hunk 3]
```diff
@@ -61,7 +61,7 @@ const configFileTemplate = `# Scaleway configuration file
 
 # To improve this tools we rely on diagnostic and usage data.
 # Sending such data is optional and can be disable at any time by setting this variable to false.
```
Comment (loicbourgois) [inline scw/config.go]: `by setting this variable to false` -> `by setting send_telemetry to false`

**Pull request merged**